tests/test_options.py

```diff
@@ -387,6 +388,45 @@ def test_opt_table_font_raises():
     assert "Either `font=` or `stack=` must be provided." in exc_info.value.args[0]
 
 
+def test_opt_row_striping():
+
+    gt_tbl_0 = GT(exibble)
+    gt_tbl_1 = GT(exibble).opt_row_striping()
+    gt_tbl_2 = GT(exibble).opt_row_striping().opt_row_striping(row_striping=False)
+
+    assert gt_tbl_0._options.row_striping_include_table_body.value == False
+    assert gt_tbl_1._options.row_striping_include_table_body.value == True
+    assert gt_tbl_2._options.row_striping_include_table_body.value == False
+
+
+def test_tab_options_striping():
+
+    gt_tbl_tab_opts = GT(exibble).tab_options(row_striping_include_table_body=True)
+    gt_tbl_opt_stri = GT(exibble).opt_row_striping()
+
+    assert gt_tbl_tab_opts._options.row_striping_include_table_body.value == True
+    assert gt_tbl_tab_opts._options.row_striping_include_stub.value == False
+
+    assert gt_tbl_opt_stri._options.row_striping_include_table_body.value == True
+    assert gt_tbl_opt_stri._options.row_striping_include_stub.value == False
+
+
+def test_tab_options_striping_snap(snapshot):
+
+    gt_tbl = GT(
+        exibble[["row", "group", "char"]].head(4), rowname_col="row", groupname_col="group"
+    ).tab_options(
+        row_striping_include_table_body=True,
+        row_striping_include_stub=True,
```

Comment:
Can you test these separately, so that we know if one options is specified, it does not set both things?

Reply:
Good idea. There is now two distinct snapshots where one tests striping only of the stub, the other only of the body cells.